x-pack/metricbeat/module/oracle/connection.go

```diff
@@ -0,0 +1,84 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package oracle
+
+import (
+	"database/sql"
+
+	"gopkg.in/goracle.v2"
+
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+
+	// Register driver.
+	_ "gopkg.in/goracle.v2"
```

Comment:
This is already loaded some lines before, do we still need this line?

Reply:
Good question! Yes. This line imports the sql go's interface driver so it has no functionality specific to Oracle while the previous import uses specific resources of the library to help parsing SID

Reply:
Actually, the blank import was used before implementing connection strings and forcing the entire package to be loaded. It can be removed as @jsoriano suggested Thanks! :slightly_smiling_face: